### DISCUSSION TITLE
Code an error message if something fails at start, in the same manner as a baseUrl server-down (the one with yellow background in Cypress UI)

### DISCUSSION THREAD

**Discussion by san-slysz** [cypress-io/cypress#18272] [Plugins] [answered]
Hi,

When starting cypress (UI) on a baseUrl (ex localhost:1234) + a unreachable server,[ we get a nice warning message in the Cypress UI. It is on a yellow background.](https://docs.cypress.io/guides/references/best-practices#With-baseUrl-set-Cypress-loads-main-window-in-baseUrl) It doesn't prevent you from actually launching a scenario at your own risk.

I'm adding code in the plugin area and would like to display a message in the same manner.
Basically for some scenario we need some env var, and if we miss one, I would rather display that, and not just prevent the whole client from running by :
```
if (env var is missing) {
          throw new Error(`Game over`)
}

```
(because this is sad)

Is there doc on that I missed? Could you give me some info or an example on how to do something in a similar manner?

**Comment by san-slysz** [comment]
@bahmutov Would ypu by any chance know how to do that?

**Comment by bahmutov** [comment, marked as answer]
Unfortunately no, I don’t think there is a way to warn via the ui

**Reply by san-slysz** [reply]
Okay. Thanks for the reply :).
